### ISSUE TITLE
[Docs] Reuse parts and remove old_docs/ from repository

### ISSUE THREAD

**Issue by nvukobratTT** [tenstorrent/tt-forge-fe#93]

**Comment by staylorTT** [comment]
Old docs has not been updated in 3 months, are we good to remove this? https://github.com/tenstorrent/tt-forge-fe/tree/main/old_docs

**Comment by nvukobratTT** [comment]
Yes, as it isn't touched from that period I think we should remove it entirely as well.

**Comment by nvukobratTT** [comment]
Here is the PR ;) 
- https://github.com/tenstorrent/tt-forge-fe/pull/1138
